javascript/javascript2/week1/homework/movie-app.js

```diff
@@ -1,21 +1,27 @@
 // JavaScript 2 Project - Movie App
 
 // Using the created API
+const url =
+  "https://raw.githubusercontent.com/mateus-britto/mateus-britto.github.io/refs/heads/main/data.json";
 const movies = []; // Variable to store the data from the API
 
-fetch(
-  "https://raw.githubusercontent.com/mateus-britto/mateus-britto.github.io/refs/heads/main/data.json"
-)
-  .then((response) => response.json())
-  .then((data) => {
-    data.forEach((movie) => {
+// Using async/await
+async function fetchData() {
+  try {
+    const response = await fetch(url);
+    if (!response.ok) {
+      throw new Error("Error fetching movies");
+    }
+    const data = await response.json();
+    for (const movie of data) {
       movies.push(movie);
-    });
-    displayMovies(movies); // Call displayMovies after movies are populated
-  })
-  .catch((error) => {
-    console.error("Error fetching movies:", error);
-  });
+    }
+    displayMovies(movies);
+  } catch (error) {
+    console.error(error.message);
+  }
+}
+fetchData();
```

Comment:
`fetchData` is an `async` function so you need to await it.

```suggestion
await fetchData();
```